Various improvements to the codebase

RedMaps opened the pull request:
*This is basically just a big refactor to keep everyones code somewhat aligned with each other.*

- Overall coding style improvements
- Increased cursor image quality
- Moved components to folders
- Changed default font
- Simplified home page svg's and navigation
- Changed default background color
- Fixed location of some files
- Added empty fanart page
- Added reroute from /home to /
- Made petals fall behind titles
- Some changes to make the site fit the design vision more closely

RedMaps committed:
Various improvements to the codebase

RedMaps commented on packages/frontend/components/Navigation/Navigation.tsx:
```diff
@@ -0,0 +1,84 @@
+                {page === item.page ? (
+                  <img
+                    src="navigation/Dot.svg"
+                    alt={item.page}
```
```suggestion
                <img src={item.iconPath alt={item.page.toString()} />
                {page === item.page ? (
                  <img
                    src="navigation/Dot.svg"
```

azuraga commented on packages/frontend/components/Navigation/Navigation.tsx:
```diff
@@ -0,0 +1,84 @@
+  CREDITS = 'credits',
+}
+
+interface NavItem {
```
Not sure to put this in a separate issue or not, but can't this, together with the formatting, be somehow made into a component? This would make the mapping around line 60 a bit less nested imo.

RedMaps replied:
Ah do you mean every nav item?

RedMaps replied:
I've decided against this for now since it seems like its more boiler plate than the trouble is worth. I've instead improved some code readability things there and extracted the array to before the component.

Mecki-Messer commented:
some of the changes you made to my code are already in place in my pr, deleting the forms and such

PS-Soundwave commented on packages/frontend/components/Footer/Footer.module.scss:
```diff
@@ -1,7 +1,8 @@
 
 .footer {
   overflow: hidden;
+  margin-top: 20rem;
```
Temporary? Seems rather large otherwise.

RedMaps replied:
Yeah temporary until we add the to the top button and to give the footer some whitespace

PS-Soundwave commented on packages/frontend/public/navigation/Corner.svg:
```diff
@@ -0,0 +1,28 @@
+<svg width="431" height="220" viewBox="0 0 431 220" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Text should not be baked into the image.

PS-Soundwave commented on packages/frontend/components/SoundBoard/SoundBoard.module.scss:
```diff
@@ -1,12 +1,12 @@
   padding: 2em 2.5em;
   margin: 2em;
-  border-radius: 1.5em;
+  border-radius: 10px;
```
Rounding looks strange particularly when zoomed out. Perhaps make it a percentage?

RedMaps replied:
I don't see anything strange but i've made the players scale better when zoomed out.

PS-Soundwave commented on packages/frontend/public/navigation/Vector.svg:
```diff
@@ -0,0 +1,3 @@
+<svg width="38" height="40" viewBox="0 0 38 40" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Unused duplicate of home icon?

nyxordinal commented on packages/frontend/components/Footer/Footer.module.scss:
```diff
@@ -39,35 +40,35 @@
-	    margin: 1px;
-	    color: $white;
-	    text-decoration: none;
-      cursor: url('../public/asacoco.cur'), default;
```
do we have a way to avoid writing relative url like this instead using something similar like `paths` in typescript ?

RedMaps replied:
I'm pretty sure there is a way, I'll look into it

RedMaps committed:
Fix minor issues

- make scss paths relative to /public
- remove unused icon
- make text on svg selectable
- change handling of audio player cards when zoomed out
- move some stuff around in Navigation
